packages/frontend/src/components/MkWatermarkEditorDialog.vue

```diff
@@ -124,7 +124,7 @@ function createStripeLayer(): WatermarkPreset['layers'][number] {
 		angle: 0.5,
 		frequency: 10,
 		threshold: 0.1,
-		black: false,
+		color: [0, 0, 0],
```

Comment:
black: false = 白なので
```suggestion
		color: [1, 1, 1],
```
だわね